Repo.rb

```diff
@@ -242,16 +243,22 @@ def write_to_disk!
     File.open(@path_to, "w") { |f| f << to_json }
   end
 
+  def recursive_write_to_disk!
+    write_to_disk!
+    subrepos.each{ |sub| sub.recursive_write_to_disk! }
```

Comment:
Space missing to the left of {.<br>Pass `&:recursive_write_to_disk!` as an argument to `each` instead of a block.